extras/userdoc/md/documentation/installation.md

```diff
@@ -1,87 +1,82 @@
 # Installation
 
+These installation instructions are for NEST 2.12 and later as well as the
+most recent version obtained from [Github](https://github.com/nest/nest-simulator). 
+Installation instructions for NEST 2.10 and earlier are provided 
+[below](#instructions-for-nest-2.10-and-earlier).
+
 ## Introduction
 
-NEST compiles and runs on most Unix-like operating systems including Linux and
-Mac OS X. For using NEST on Microsoft Windows, see [below](installation.md#windows).
-The installation instructions here should work on all recent versions of Debian
-or Ubuntu. For more generic installation instructions, see the files README and
-INSTALL in the top source directory.
+NEST compiles and runs on most Unix-like operating systems. 
+The installation instructions here should work on recent versions of Ubuntu or 
+Debian, while additional instructions for 
+[OS X/macOS are given below](#macos). 
```

Comment:
Fixed. Based [on this information](https://gist.github.com/asabaylus/3071099#gistcomment-1593627) I also updated the link to the NEST 2.10 documentation.